cdap-docs/reference-manual/source/characters.rst

```diff
@@ -12,10 +12,10 @@ Supported Characters
 
 The Cask Data Application Platform (CDAP) has naming conventions for different components of CDAP.
 
-Streams
--------
+Streams and Metadata
```

Comment:
I notice that we don't currently say in the docs that ``$`` is allowed for dataset names. Should that be added?